### PR TITLE
Update auth URL in CLI docs

### PR DESCRIPTION
**Pull request by karlrl**

### CODE REVIEW

File: content/101/authenticating-cli-tools.md [hunk 1]
```diff
@@ -38,5 +38,4 @@ list of variables and what their corresponding values should be:
  email address)
  - `OS_PASSWORD` - Your Cloud A password
  - `OS_TENANT_NAME` - Typically the same as OS_USERNAME
- - `OS_AUTH_URL` - Set to http://api.cell01.clouda.ca:5000/v2.0/ Cloud A
- Windows Environment
+ - `OS_AUTH_URL` - Set to https://keystone.ca-ns-1.clouda.ca:8443/v2.0
```
Comment (karlrl) [inline content/101/authenticating-cli-tools.md]: This is the auth URL for `ca-ns-1`. Would it be better to put `<region-name>` or similar in the example URL?

Comment (karlrl) [inline content/101/authenticating-cli-tools.md]: Actually, I should also add the `OS_REGION_NAME` to this list.

**Comment by karlrl** [comment]
Happy to squash commits if/when this looks good to merge.

**Comment by thurloat** [comment]
LGTM, good to merge.

**Pull request merged**